lib/drivers/common/prepare.js

```diff
@@ -47,13 +47,19 @@ function spawn(dir, env, cmd, callback) {
   var options = {
     cwd: dir,
     env: _.extend({}, process.env, env),
-    stdio: 'inherit',
+    stdio: debug.enabled ? ['ignore', 'pipe', 'pipe'] : 'ignore',
```

Comment:
Does stderr from npm provide enough context on its own? I figured it would need to be combined with stdout to be useful.